wrappers/jquery/mustache.js.pre

```diff
@@ -5,5 +5,6 @@ Shameless port of a shameless port
 See http://github.com/defunkt/mustache for more info.
 */
  
+ // This seems useless and not valid js syntax @TODO
```

Comment:
It's actually correct js syntax, and not useless – it's part of the jquery wrapper generator, the header part.